### DISCUSSION TITLE
OSx Monterey broke my LAN

### DISCUSSION THREAD

**Discussion by controlfix87** [osy86/HaC-Mini#956] [Q&A] [answered]
Hi :)
I've just updated to OSx Monterey, 
I did an NVRAM reset right after. 
Installed the HaCMini 3.1.2 before and after the os update, but nothing..
LAN is disconnected even when the cable worked just fine a moment before, but now, nothing.

Also, I don't have any audio AT ALL... I did have it before..

Please help on that.

**Comment by controlfix87** [comment]
Anyone can help on that?

**Comment by osy** [comment, marked as answer]
See #947, you can wait for Small Tree to update the drivers, downgrade to Big Sur, or use the other LAN port.
